### DISCUSSION TITLE
BTT SKR Mini E3 not connecting

### DISCUSSION THREAD

**Discussion by MsaterSgnt** [feelfreelinux/octo4a#127] [Q&A]
I'm trying to setup octo4a, but it keeps telling me that the printer is not connected. After looking at the FAQ, I saw a section for that there, but I'm not sure which option to use or how to implement it.

**Comment by Ursa80** [comment]
I have the same issue with my SKR mini E3 1.2

**Reply by MsaterSgnt** [reply]
My issue was a faulty cable, try that.

**Reply by Ursa80** [reply]
Which serial port settings did you set? SKR board does not work with any settings. But old board from Ender-3 works fine!
